pinot-controller/src/main/java/org/apache/pinot/controller/api/resources/PinotTableRestletResource.java

```diff
@@ -712,6 +719,37 @@ public String getTableState(
     }
   }
 
+  @GET
+  @Produces(MediaType.APPLICATION_JSON)
+  @Authenticate(AccessType.UPDATE)
+  @Path("/rebalanceStatus/{jobId}")
+  @ApiOperation(value = "Gets detailed stats of a rebalance operation",
+          notes = "Gets detailed stats of a rebalance operation")
+  public ServerRebalanceJobStatusResponse rebalanceStatus(@ApiParam(value = "Rebalance Job Id", required = true)
```

Comment:
I feel that it's useful to add an API to list ongoing rebalance. It's useful after users invoke the rebalance API but they don't save it any where. Without the id, they cannot get the status anymore.

Reply:
Yes makes sense. We already have an API to retrieve all jobs for a table. This takes in a jobType as well and reload is one of them today. I'll add rebalance as another type and we can leverage the same API. 
@Path("table/{tableName}/jobs")
fyi @mayankshriv

Reply:
Enhanced the following API to show the rebalance Jobs as well for a table. This will list out rebalanceJobIds to choose from, to see status. 
**path("table/{tableName}/jobs")**

@saurabhd336  - Could you please take a look as well, since I've leveraged your api?